docs/nydus.md

```diff
@@ -0,0 +1,26 @@
+# Lazy-pulling using Nydus Snapshotter
+
```

Comment:
```markdown
| :zap: Requirement | nerdctl >= 0.22 |
|-------------------|----------------|
```

Reply:
(And kernel version, maybe?)

Reply:
@AkihiroSuda Thanks for the reply, the kernel version is not mandatory if users don't use the in-kernel EROFS backend (added below).